VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.html

```diff
@@ -6,8 +6,8 @@
   </p>
   <hr />
   <a *ngIf="participant.isInHearing()" (click)="toggleMuteParticipant()">{{ getMutedStatusText() }}</a>
-  <a *ngIf="participant.isWitness && !participant.isInHearing()" (click)="callWitnessIntoHearing()">{{'judge-context-menu.call-witness' | translate}}</a>
-  <a *ngIf="participant.isWitness && participant.isInHearing()" (click)="dismissWitnessFromHearing()">{{'judge-context-menu.dismiss-witness' | translate}}</a>
+  <a *ngIf="(participant.isWitness || participant.isQuickLinkObserver) && !participant.isInHearing()" (click)="callWitnessIntoHearing()">{{'judge-context-menu.call-witness' | translate}}</a>
```

Comment:
As this is going to grow with (for QL participant) and all the checks are on the participant object can we have either a function in the TS that takes the participant and returns a boolean or another getter on the participant model like canBeDismissed.

Reply:
Maybe we should have a term for the type of participant role that can be called and dismissed by the judge. I can't think of a word that encapsulates that right now though.

isSummonable?

Reply:
@robertbennett1998  I will refactor that in the story 7905. This was dealing with only observer, I kept it as it is.